lib/ansible/module_utils/network/cnos/cnos.py

```diff
@@ -2253,37 +2277,35 @@ def vlanConfig(
     else:
         value = checkSanityofVariable(deviceType, "vlan_id", vlanArg1)
         if(value == "ok"):
-            retVal = createVlan(obj, deviceType, "(config-vlan)#",
-                                timeout, vlanArg1, vlanArg2, vlanArg3,
-                                vlanArg4, vlanArg5)
+            retVal = createVlan(module, '(config-vlan)#', None)
             return retVal
         else:
             value = checkSanityofVariable(
                 deviceType, "vlan_id_range", vlanArg1)
             if(value == "ok"):
-                retVal = createVlan(obj, deviceType, "(config-vlan)#",
-                                    timeout, vlanArg1, vlanArg2, vlanArg3,
-                                    vlanArg4, vlanArg5)
+                retVal = createVlan(module, '(config-vlan)#', None)
                 return retVal
             retVal = "Error-133"
             return retVal
 
     # debugOutput(command)
-    command = command + "\n"
-    # debugOutput(command)
-    retVal = retVal + waitForDeviceResponse(command, prompt, timeout, obj)
+    cmd = [{'command': command, 'prompt': None, 'answer': None}]
+    retVal = retVal + str(run_cnos_commands(module, cmd))
     return retVal
 # EOM
 
 
-def vlanAccessMapConfig(
-        obj, deviceType, prompt, timeout, vlanArg3, vlanArg4, vlanArg5):
-    retVal = ""
+def vlanAccessMapConfig(module, prompt, answer):
+    retVal = ''
     # Wait time to get response from server
-    timeout = timeout
-    command = ""
+    timeout = 2
```

Comment:
Again with timeout

Reply:
Not relevant now, will remove. But persistence connection dont have a provison to specify timeout for each command , A common 30 sec time out is only there I feel.

Reply:
Removed